internal/store/engine_state.go

```diff
@@ -453,7 +453,8 @@ type ManifestState struct {
 	LiveUpdatedContainerIDs map[container.ID]bool
 
 	// We detected stale code and are currently doing an image build
-	NeedsRebuildFromCrash bool
+	//NeedsRebuildFromCrash bool
+	CrashTime time.Time
```

Comment:
It's actually gone in the latest revision 😶 

We don't have similar info for runtime so after talking with Maia we decided to just punt on it for the moment since it's not critical for initial use case, but should be relatively straightforward to add in, just some additional bookkeeping around runtime/deploy similar to what is already done for build